### ISSUE TITLE
Bug: There was an error adding new feed: Item not found

### ISSUE THREAD

**Issue by flethj** [omnivore-app/omnivore#3672]
I'm running into an issue when trying to import an RSS feed. 

Feed:
https://openrss.org/https://www.youtube.com/@ByteByteGo/videos

Error:
"There was an error adding new feed: Item not found"

The feed appears to exist and it can be imported into RSS readers (I tried Reeder and it worked). However, it does not work in Omnivore. I tried both the web and iOS versions.

**Comment by jacksonh** [comment]
Hey @flethj are you directly adding that URL? It looks like an HTML page to me but maybe its meant to change based on accepted content type?

**Comment by flethj** [comment]
Hello, I tried to add exactly this url. The returned content probably depends on something like the accepted content type. Chrome shows the HTML page but for example Safari recognizes the RSS feed and asks whether to open it in an RSS reader.

The weird thing is that I've successfully used openrss.org in combination with YouTube channels in Omnivore before. Exactly as above but with a different channel. A different RSS client is also (still) able to see the feed with this exact url, which is why I suspect that something in Omnivore broke.

**Comment by jacksonh** [comment]
Yeah its likely because we added some extra accepted content types because a few other feeds required things like HTML be accepted, probably due to misconfigured load balancers.

**Comment by niksart** [comment]
I get the same error adding this feed:

https://direct.mit.edu/rss/site_1000093/1000049.xml

Should I open a new issue?

**Comment by jacksonh** [comment]
> I get the same error adding this feed:
> 
> https://direct.mit.edu/rss/site_1000093/1000049.xml
> 
> I guess the problem has to do with the presence of an underscore in the link. Should I open a new issue?

```
curl https://direct.mit.edu/rss/site_1000093/1000049.xml 
<html><body><h1>403 Forbidden</h1>
Request forbidden by administrative rules.
</body></html>
```

looks like they have some blocking of that feed.

**Comment by niksart** [comment]
It seems that the problem is the user agent "curl" that is blocked by them:
```
curl -A "curl" "https://direct.mit.edu/rss/site_1000093/1000049.xml"
<html><body><h1>403 Forbidden</h1>
Request forbidden by administrative rules.
</body></html>
```

Any other string works:

```
curl -A "qwertyuiop" "https://direct.mit.edu/rss/site_1000093/1000049.xml"
```

**Comment by jacksonh** [comment]
Yeah i suspect they are also blocking some IPs, i can add this feed fine in local development but can't from our backend.

**Comment by fabianlandwehr1** [comment]
Hello, are there any plans to change the accepted content types again (so that openrss.org works again)?

**Comment by CopyPasteFail** [comment]
same problem for me here:
https://www.geektime.co.il/feed/

**Comment by micmalti** [comment]
I'm facing the same issue with the DistroWatch RSS feeds.

**Comment by jacksonh** [comment]
> Same problem for me here:
> 
>     * https://www.geektime.co.il/feed/
> 
> 
> `$ curl https://orikatz.wordpress.com/feed/`
> 
> `<?xml version="1.0" encoding="UTF-8"?><rss version="2.0" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:wfw="http://wellformedweb.org/CommentAPI/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:sy="http://purl.org/rss/1.0/modules/syndication/" xmlns:slash="http://purl.org/rss/1.0/modules/slash/" xmlns:georss="http://www.georss.org/georss" xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#" xmlns:media="http://search.yahoo.com/mrss/" > <channel>............`
> 
> The URL works in feedly.

I get this: https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fwww.geektime.co.il%2Ffeed%2F

**Comment by jacksonh** [comment]
> I'm facing the same issue with the DistroWatch RSS feeds.

Can you give an example URL you are using?

**Comment by micmalti** [comment]
Sure. Here's one: https://distrowatch.com/news/dw.xml

**Comment by stevenrobertson** [comment]
Same issue with https://www.science.org/digital-feed/pipeline

**Comment by makamto** [comment]
Same issue with https://www.inmediahk.net/full/feed

**Comment by makamto** [comment]
Same issue with https://www.rfa.org/cantonese/news/rss2.xml

**Comment by sywhb** [comment]
> I'm running into an issue when trying to import an RSS feed.
> 
> Feed: https://openrss.org/https://www.youtube.com/@ByteByteGo/videos
> 
> Error: "There was an error adding new feed: Item not found"
> 
> The feed appears to exist and it can be imported into RSS readers (I tried Reeder and it worked). However, it does not work in Omnivore. I tried both the web and iOS versions.

Hey, this seems to be working now

**Comment by sywhb** [comment]
> https://distrowatch.com/news/dw.xml

This seems to be working now

**Comment by sywhb** [comment]
Hey @jacksonh , I have tested these failed RSS feeds and I get 403 for all of them. Even when I open https://www.inmediahk.net/full/feed in the browser, I sometimes have to click on the Cloudflare captcha which makes me wonder if we should scrape the feed content in puppeteer.

It might be a little helpful but still need to figure out a better way to bypass the captcha with proxy and fingerprint
